pkg/catalogsourceconfig/clusterrolebindingbuilder.go

```diff
@@ -0,0 +1,66 @@
+package catalogsourceconfig
+
+import (
+	"github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+	rbac "k8s.io/api/rbac/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterRoleBindingBuilder builds a new ClusterRoleBinding object.
+type ClusterRoleBindingBuilder struct {
+	crb rbac.ClusterRoleBinding
+}
+
+// ClusterRoleBinding returns a ClusterRoleBinding object.
+func (b *ClusterRoleBindingBuilder) ClusterRoleBinding() *rbac.ClusterRoleBinding {
+	return &b.crb
+}
+
+// WithTypeMeta sets basic TypeMeta.
+func (b *ClusterRoleBindingBuilder) WithTypeMeta() *ClusterRoleBindingBuilder {
+	b.crb.TypeMeta = meta.TypeMeta{
+		Kind:       "ClusterRoleBinding",
+		APIVersion: "v1",
+	}
+	return b
+}
+
+// WithMeta sets basic TypeMeta and ObjectMeta.
+func (b *ClusterRoleBindingBuilder) WithMeta(name string) *ClusterRoleBindingBuilder {
+	b.WithTypeMeta()
+	if b.crb.GetObjectMeta() == nil {
+		b.crb.ObjectMeta = meta.ObjectMeta{}
+	}
+	b.crb.SetName(name)
+	return b
+}
+
+// WithOwner sets the owner of the ClusterRoleBinding object to the given owner.
+func (b *ClusterRoleBindingBuilder) WithOwner(owner *v1alpha1.CatalogSourceConfig) *ClusterRoleBindingBuilder {
+	b.crb.SetOwnerReferences(append(b.crb.GetOwnerReferences(),
+		[]meta.OwnerReference{
+			*meta.NewControllerRef(owner, owner.GroupVersionKind()),
+		}[0]))
```

Comment:
nit: Why make a slice of meta.OwnerReference and then immediately reference the first element of it? Why not just initialize the NewControllerRef directly?

Reply:
`ClusterRoles` and `ClusterRoleBindings` are no longer needed. I forgot to remove the files. I will do so now.